imports/plugins/core/accounts/client/templates/profile/userOrdersList.html

```diff
@@ -0,0 +1,11 @@
+<template name="userOrdersList">
+  {{#if hasData data=data}}
+    <div>
+      {{> React completedOrders }}
+    </div> 
+  {{else}}
+    <div class="alert alert-info">
```

Comment:
The component should be handling both the found and not found cases.

Reply:
Okay. I'll create functions to render found and not found cases in the component.